cookbooks/wazuh_agent/recipes/agent.rb

```diff
@@ -85,14 +85,12 @@
   args << ' -k ' + agent_auth['key']
 end
 
-if agent_auth['password']
```

Comment:
This piece of code is duplicated on [L71](https://github.com/wazuh/wazuh-chef/pull/119/files#diff-fffce9a3886a2b4d0a739df924002b58e24e23fffb62c21e0f54ab2aec0b6fc2L71).